vagrant_repo/data_bags/cerner_splunk/passwords_keys.json

```diff
@@ -15,20 +15,28 @@
   "c1_slave1": "CyiHOyM+AINlsSDukR0AYX7BNfT7Z9jSsLtMYXcf+9lCsxShr98ISfoUOfGX\nnfZGWC4dT1PttRe4WikbFlEheG06s6vJRODCNFY7UEsxvamAbgC6J5VRYYVB\nceXUCNLDQOS42riTUSgYAEWFeXbmcCiUbztJFqDsFNjFz1LvMTncQzvZ7zEj\nDnA0n/Oaw2Rlrjcgj2BCNmYb3bP8fQQdq4duiArIqwgSOnylVn2qUI2C5/p8\nMYeQ2PwKw6wOwe7ztThqhwZF1KDctIzORCFAayfOGvfg+3ub8Ps/tsXY1+Pq\ngpYM7ucvpO0TYnFdszvBEHWneJuevkHURH0+JtRmog==\n",
   "c1_slave2": "Yaf4kjxtU/T74hfQmekw4IiBrYV1jaZO3spXfT2NFCs2cu4R2Cr1KcXyTs9p\n2d2DXjwQkZUDT0T3PIBaw4x8TsmYlVwvXOxfgY7cLJXsW5b6cqxlS2ApajqI\nMH3AjBBBC7m5DD7M+zT1FKlJmfqxLbO8QxQumYSP8SZ8t27LCb6xoTSWXEfT\n2mGjwSqRDkifs/YbRzYeDDBkSv80gsn0Wu7uNGay0NM+nwuTKbnWFAw3zHNJ\nq37He9p2/XLJ1XODhzxXeOzSSHVDN4wW/iNZmuyVgQWvcA4MiFJgzISq6jI7\nV3FOjep0FZagVQgMr4cQ57hLS3yEZsBvyeQk3lTDKA==\n",
   "c1_slave3": "MTRZlzKg97eZSXTrcI9zDHObhPKqvGJpfHRsGFYJYOBCSdZMGjPeptDAtqhq\nucSHTsmEgDtBoGO7q2d0X3rYsF5yPDAKUBkzOctlGyr2OiNE0wv0liRkJnWB\nZ7IkknCoZsmmUIm60lWR7krbGYqgt9G7+NH+PbCI26Boe2PpvN5xFIDBoPzA\nJOPqskir3NjxGflOQNwfuLGqAiH3EZAQ3b2dikLeheS0SJEPdmWYZIBRfnAt\n8Dn+ahPVZTW907w7fJHDdPFKFB5REOZGQbwiM+bfBJXDaVgKB2A+QsZmVfin\nuObXFe2F2CdyN1XKsR5hx7lBlQEhecOrSMClbBOJHw==\n",
-  "knife_workstation": "KrYDUXBt7fXMcGYhPwoW4cj0gklxj+g1cq7uWeORF4ifVd89G+777s/jslZi\nSxIUGt/kq6jmPlQaW6CnR2hnIIyVerwLerGvKhtqqsQ0mAqlUVTjdvSNEeMh\nBAK60MehdL/mxtsJZJngqyiXv9KoACyxPC5fBfTjYz857cj2pjyNFUClwWax\nnn9F3MirTvEyrzc/orhrlexZps0kOrc9Q4zUNNrm7WsvOi2845GW91OrvD+v\nEFrD0gr2UIQ/zfz7siF+UMHvr/Hdurv69lUY3TCYIE4FuYxclj8VxGMEhAt7\n3I0Tztzio4BgGtAGk5lhP9mc4RYXtXBJLCFf3ia/ew==\n",
+  "knife_workstation": "CVfLEyiNMccDYrReoT7GRDhEkps+eXRTquUFB87hS/FKu7RRVm7yfN5dp61A\n4Ap9r4ykhqapByT02iAZnbB5XqmtZruIrbkZkKrr+iCPrKMzMMs6PtQBGB+g\nMKkWcdJXwAc01R2s2V8BL1YPF/g3kH7izhGprzzjxBwiIVgBzQIMyYGGvVYh\njRQLv/9P1WbEAQNt8tjZhyxTMv5TL83VAJ7+kxAc/G7+i+5kWz2hMNX9RWve\n3ime+7isbTH1Z1jo9KGdUQ+kE0MoV2yaR5yPxUHtku181ww2vLIMd3SPjFdv\nTloeN57wEZ0Rrx4wDQuxlUH7jlBZFWD5MR1NtaBePQ==\n",
   "s_standalone": "zwiC85yUmPJ0q+JaQzrQ6Ug2UsW7F/fBbLiwC26g58i/zbNlS+rvOggsivDV\n0YUs1UZ8rzv/fvM5vfzr3gPLuUYBt2/t9+88GJgcR3zR09E0K77rR0WWsSuo\nT4XAU4pWJM6TsANjnlUCkmgQabHwDhmS3cFxCk3itKiwft0vHWKowj6ObkA4\nGO2udCCX7LrRe8v8o4dLl20Rom3VFNYPD5YlvnMsnrSHoRAsw9EjucLienQ3\nKb4Nvk+DRs2gX6k908G7Zp3tSxx/zHvwmAZrImweNIn5c5F9GzSlVY0rvHo1\n5MZ1Y8IfvA0LqEoMVTLn8ZMY7jivA5ysFVef4Ix3KQ==\n",
   "s_license": "iTEgtfQFH0AuFpdYyoptYSNUkcILZEHS0l8pdGtb+ImaHxuA+rhL1REX+IJK\nA1SxTlTaLgnMC6lre6a2wykWl9H9sXhUUMZGgAwHfuC9vzfWA8s+tFKW58bZ\ncM/2wsqX8e3TNO/lrf0awmvmBKnI1ZErjBzlWcwQFTEl2GycPK0gVnHFz1LZ\nut93D7dT+tJIRR/bpJ5Sx/XtbcWXPcVVlTR02MoHeFA3CR/xnvtMV2qIKymo\noBZo9Emzd3soAwzgoHhWZKTIYrRvtBuRB6IOU1RYL+Jyus08zfqgzSaZzJsg\ngQkPm5s/wuxdPUqwMFLfmAX9++bpaFQ7YdIkNnfnww==\n",
```

Comment:
Sure enough... using code similar to https://gist.github.com/acharlieh/88e752e9e6b71980778530debe993715 shows that they all decrypt to the same symmetric key value. Looking further into it, the answer is the padding function, where random numbers are included to make the plaintext longer: https://tools.ietf.org/html/rfc2437#section-7.2.1